[INFO] Scanning for projects...
[INFO] 
[INFO] --------------< cws_2020:cws_spareparts_aceptance_tests >---------------
[INFO] Building cws_spareparts_aceptance_tests 0.0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ cws_spareparts_aceptance_tests ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 0 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:compile (default-compile) @ cws_spareparts_aceptance_tests ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ cws_spareparts_aceptance_tests ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 19 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:testCompile (default-testCompile) @ cws_spareparts_aceptance_tests ---
[INFO] Nothing to compile - all classes are up to date
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ cws_spareparts_aceptance_tests ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running uo.ri.cws.application.service.RunCucumberTests

Scenario: Update an existing provider                                        # test/uo/ri/cws/application/service/providers/Update.feature:6
  Given a provider with nif '123456789z'                                     # uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(java.lang.String)
      java.lang.NullPointerException
	at uo.ri.cws.application.service.common.ProviderSteps.register(ProviderSteps.java:124)
	at uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(ProviderSteps.java:45)
	at ✽.a provider with nif '123456789z'(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:7)

  When I update the provider with nif '123456789z' for all its fields        # null
  Then the provider results updated for all the fields except the nif and id # uo.ri.cws.application.service.common.ProviderSteps.theProviderResultsUpdatedForAllTheFields()

Scenario: Try to update an non existing provider                       # test/uo/ri/cws/application/service/providers/Update.feature:11
  When I try to update the provider with nif 'does-not-exist-provider' # uo.ri.cws.application.service.common.ProviderSteps.iTryToUpdateAProviderWithTheNif(java.lang.String)
      io.cucumber.java.PendingException: TODO: implement me
	at uo.ri.cws.application.service.common.ProviderSteps.iTryToUpdateAProviderWithTheNif(ProviderSteps.java:92)
	at ✽.I try to update the provider with nif 'does-not-exist-provider'(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:12)

  Then an error happens with an explaining message                     # uo.ri.cws.application.service.common.CommonSteps.anErrorHappensWithAnExplainingMessage()

Scenario: Try to update to repeated fields                                          # test/uo/ri/cws/application/service/providers/Update.feature:15
  Given the providers with                                                          # uo.ri.cws.application.service.common.ProviderSteps.TheProvidersWith(io.cucumber.datatable.DataTable)
      io.cucumber.java.PendingException: TODO: implement me
	at uo.ri.cws.application.service.common.ProviderSteps.TheProvidersWith(ProviderSteps.java:143)
	at ✽.the providers with(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:16)

  When I try to update the provider with nif '5678x' to <name>, <email> and <phone> # uo.ri.cws.application.service.common.ProviderSteps.iTryToUpdateTheProviderWithNifForNameEmailAndPhone(java.lang.String,io.cucumber.datatable.DataTable)
  Then an error happens with an explaining message                                  # uo.ri.cws.application.service.common.CommonSteps.anErrorHappensWithAnExplainingMessage()

Scenario Outline: Try to update with wrong parameters                                                     # test/uo/ri/cws/application/service/providers/Update.feature:32
  Given a provider with nif '123456789z'                                                                  # uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(java.lang.String)
      java.lang.NullPointerException
	at uo.ri.cws.application.service.common.ProviderSteps.register(ProviderSteps.java:124)
	at uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(ProviderSteps.java:45)
	at ✽.a provider with nif '123456789z'(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:26)

  When I try to update the provider with nif '123456789z' to "", "provider-new@email.com" and "new-phone" # null
  Then an error happens with an explaining message                                                        # uo.ri.cws.application.service.common.CommonSteps.anErrorHappensWithAnExplainingMessage()

Scenario Outline: Try to update with wrong parameters                                       # test/uo/ri/cws/application/service/providers/Update.feature:33
  Given a provider with nif '123456789z'                                                    # uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(java.lang.String)
      java.lang.NullPointerException
	at uo.ri.cws.application.service.common.ProviderSteps.register(ProviderSteps.java:124)
	at uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(ProviderSteps.java:45)
	at ✽.a provider with nif '123456789z'(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:26)

  When I try to update the provider with nif '123456789z' to "new-name", "" and "new-phone" # null
  Then an error happens with an explaining message                                          # uo.ri.cws.application.service.common.CommonSteps.anErrorHappensWithAnExplainingMessage()

Scenario Outline: Try to update with wrong parameters                                                    # test/uo/ri/cws/application/service/providers/Update.feature:34
  Given a provider with nif '123456789z'                                                                 # uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(java.lang.String)
      java.lang.NullPointerException
	at uo.ri.cws.application.service.common.ProviderSteps.register(ProviderSteps.java:124)
	at uo.ri.cws.application.service.common.ProviderSteps.aProviderWithNif(ProviderSteps.java:45)
	at ✽.a provider with nif '123456789z'(file:///C:/Users/alb/git/cws_lab_2020/cws_spareparts_e2e_tests/test/uo/ri/cws/application/service/providers/Update.feature:26)

  When I try to update the provider with nif '123456789z' to "new-name", "provider-new@email.com" and "" # null
  Then an error happens with an explaining message                                                       # uo.ri.cws.application.service.common.CommonSteps.anErrorHappensWithAnExplainingMessage()
[ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0, Time elapsed: 2.334 s <<< FAILURE! - in uo.ri.cws.application.service.RunCucumberTests
[ERROR] Update a provider.Update an existing provider  Time elapsed: 0.635 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] Update a provider.Try to update an non existing provider  Time elapsed: 0.049 s  <<< ERROR!
io.cucumber.java.PendingException: TODO: implement me

[ERROR] Update a provider.Try to update to repeated fields  Time elapsed: 0.071 s  <<< ERROR!
io.cucumber.java.PendingException: TODO: implement me

[ERROR] Update a provider.Try to update with wrong parameters #1  Time elapsed: 0.052 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] Update a provider.Try to update with wrong parameters #2  Time elapsed: 0.11 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] Update a provider.Try to update with wrong parameters #3  Time elapsed: 0.119 s  <<< ERROR!
java.lang.NullPointerException

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   TODO: implement me
[ERROR]   TODO: implement me
[ERROR]   null
[ERROR]   null
[ERROR]   null
[ERROR]   null
[INFO] 
[ERROR] Tests run: 6, Failures: 0, Errors: 6, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.538 s
[INFO] Finished at: 2020-09-17T20:36:54+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project cws_spareparts_aceptance_tests: There are test failures.
[ERROR] 
[ERROR] Please refer to C:\Users\alb\git\cws_lab_2020\cws_spareparts_e2e_tests\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
